backend/app.ts

```diff
@@ -0,0 +1,87 @@
+import * as dotenv from 'dotenv';
+import express from 'express';
+import cors from 'cors';
+import morgan from 'morgan';
+import session from 'express-session';
+import passport from 'passport';
+import googleOAuthRouter from './routes/googleOAuth';
+import testRouter from './routes/testRoute'
+import prisma from './utils/prismaClient';
+import loginAuth from './routes/loginAuth'
+import googleStrategy from 'passport-google-oauth20';
+dotenv.config();
+const GoogleStrategy = googleStrategy.Strategy;
+
+const db = prisma;
+const app = express();
+
+app.use(morgan("dev"));
+app.use(cors());
+app.use(express.json());
+
+const secretcode = process.env.SESSION_SECRET as string;
+const clientID = process.env.GOOGLE_CLIENT_ID as string;
+const clientSecret = process.env.GOOGLE_CLIENT_SECRET as string;
```

Comment:
no change needed but it's common to put all of your ENV vars in a `config.ts` file that gets imported to files as you need it.